sdk/storage/blobs/README.md

```diff
@@ -1,7 +1,114 @@
-# Azure SDK Storage Blobs Library
+# Azure SDK Storage Blobs Library for C
 
 Azure Storage Blobs library (`az_storage_blobs`) provides abstractions, and helpers for communicating with Azure Storage Blobs in the C programming language. This library follows the Azure SDK Design Guidelines for Embedded C.
 
-The library allows clients to communicate with Azure Storage Blobs.
+Use the Azure SDK Storage Blobs library for C to communicate with Azure Storage Blobs.
```

Comment:
I wouldn't use the word "communicate"; I'd instead say: Use the Azure SDK Storage Blobs library for C to work with Azure Storage Blobs.